### ISSUE TITLE
Unable to Add Manifest.

### ISSUE THREAD

**Issue by qatoolist** [RedHatInsights/subscription-central-ui#36]
**Steps to reproduce :**

1. Login to : https://console.stage.redhat.com/beta
2. Navigate to Manifests using "Find an App or Service" -> "Manifests"
3. Click Create Manifests button
4. Provide Details [Anand-Test1, Satellite 6.9 ] and Save
    **Expected -** A new manifest should be created.
    **Actual -** Manifest is not Created
5. Close the Create Manifests View
6. Click on "Manifests" Menu in LHS Navigation panel
7. Verify Manifests page
    **Expected -** Manifests page should be rendered correctly
    **Actual -** Manifest page appears with errors

**Environment:** Stage/Beta

**Comment by joeldmyers** [comment]
As to parts 1-4 -- as discussed, this appears to be related to a 403 error due to a misconfiguration of permissions for the test user involved. The error message in Splunk is "account does not have a Satellite subscription". Hence, the create manifest form _should_ fail here. Separately there is an issue in the toast error message not displaying correctly; that has been resolved in [this PR](https://github.com/RedHatInsights/subscription-central-ui/pull/34) and merged; it should appear in stage by tomorrow morning at the latest. Let me know if that hasn't been deployed to stage by then.

Separately, as far as 6-8, it appears this bug occurs no matter what one does within the app -- if you navigate to manifests page and then click that side nav link again, it will show an error. There's a good chance this is due to the updates from c.rh.c team with the new side nav architecture. I created a [new issue](https://github.com/RedHatInsights/subscription-central-ui/issues/37) for this and will follow up separately, as I think this should be handled separately from this story.

Let me know if anything further needs to be done on this front.
